tests/test_yask.py

```diff
@@ -452,8 +453,9 @@ def test_acoustic_w_src_w_rec(self, model, eqn, m, damp, u, src, rec):
 
         # The expected norms have been computed "by hand" looking at the output
         # of test_adjointA's forward operator w/o using the YASK backend.
-        exp_u = 153.25
-        exp_rec = 212.03
+        exp_u = 159.94
```

Comment:
No idea why these values changed in this PR but it did.

Reply:
anything to do with resampling?

Reply:
Nothing is resampled here. Something changed somewhere but can't find out what

Reply:
that's bad. We should find out

Reply:
might it be because the "number of timesteps" calculation has changed (in TimeAxis)?

Anyway, I get we are calculating the right thing, so that's fine